### PR TITLE
modify VAP docs

### PR DESCRIPTION
**Pull request by MariamFahmy98**
## Related issue #
None

## Proposed Changes
This  PR does the following:
1. Modify the VAP docs.
2. Fix the container flags section.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by chipzoller** [comment]
I removed the numbers from the container flags list because it wasn't very useful and was the source of difficulty in accepting multiple PRs which modified the same content, leading to merge conflicts.

**Comment by MariamFahmy98** [comment]
> I removed the numbers from the container flags list because it wasn't very useful and was the source of difficulty in accepting multiple PRs which modified the same content, leading to merge conflicts.

But it seems to be messed up, what about add them to a table instead?

**Comment by chipzoller** [comment]
> > I removed the numbers from the container flags list because it wasn't very useful and was the source of difficulty in accepting multiple PRs which modified the same content, leading to merge conflicts.
> 
> But it seems to be messed up; what about adding them to a table instead?

Damn it. I clearly didn't even look at my own preview.

Let's put some line breaks `<br>` between them.

File: content/en/docs/Writing policies/validating-admission-policies/generate-validating-admission-policies.md [hunk 1]
```diff
@@ -14,7 +14,35 @@ Because Kyverno can be used to generate ValidatingAdmissionPolicies and their bi
 
 To generate ValidatingAdmissionPolicies, make sure to:
 1. enable `ValidatingAdmissionPolicy` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
-2. enable either `admissionregistration.k8s.io/v1alpha1` or `admissionregistration.k8s.io/v1beta1` API depending on the version of Kubernetes you are running.
+2. for 1.27, enable `admissionregistration.k8s.io/v1alpha1` API, and for 1.28 enable both `admissionregistration.k8s.io/v1alpha1` and `admissionregistration.k8s.io/v1beta1` API.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/validating-admission-policies/generate-validating-admission-policies.md:17]: I'm pretty sure for 1.28 you don't have to enable the v1alpha1 version.

Comment (MariamFahmy98) [reply content/en/docs/Writing policies/validating-admission-policies/generate-validating-admission-policies.md:17]: In the codebase, we generate the `v1alpha1` of VAPs. That's why it needs to be enabled along with `v1beta1`.

Comment (chipzoller) [reply content/en/docs/Writing policies/validating-admission-policies/generate-validating-admission-policies.md:17]: Ok

**Review by chipzoller — APPROVED** [review]

**Pull request merged**